debian/changelog

```diff
@@ -1,8 +1,102 @@
-cloud-init (22.2-0ubuntu1~22.10.2) UNRELEASED; urgency=medium
+cloud-init (22.2-64-g1fcd55d6-0ubuntu1~22.10.1) kinetic; urgency=medium
 
   * d/control: add python3-debconf as Depends and Build-Depends
-
- -- Alberto Contreras <alberto.contreras@canonical.com>  Mon, 20 Jun 2022 10:54:54 +0200
+  * d/gbp.conf d/gbp_format_changelog:
+    + git-buildpackage customization for debian/changelog generation
+  * New upstream snapshot.
+    + tests: mock dns lookup that causes long timeouts (#1555)
+    + tox: add unpinned env for do_format and check_format (#1554)
+    + cc_ssh_import_id: Substitute deprecated warn (#1553)
+      [Alberto Contreras]
+    + Remove schema errors from log (#1551) (LP: #1978422) (CVE-2022-2084)
```

Comment:
Our new-upstream-snapshot formatting didn't handle the CVE: <ID> footer and append it. I have taken the liberty to add that suffix to this entry.